src/PMD/PMD.jl

```diff
@@ -60,9 +74,144 @@ function _get_type(str)
         return String
     elseif str == "DATE"
         return Dates.Date
+    elseif str == "REFERENCE"
+        return Ptr{Nothing}
```

Comment:
Usually `REFERENCE` is mapped as a String

Reply:
This was only added to parse the file.
I am no sure what should be the type.
I dont understand why it is usually a string. Have examples?

Reply:
Only from OptFolio, I can try to convert some case and send it to you.

Reply:
ping

Reply:
sending you internally